.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,6 +7,8 @@
 - [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
 - [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
 - [ ] Gallery example in `./doc/examples` (new features only)
+- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
```

Comment:
Do we really want a benchmark for every new piece of functionality?

Reply:
@stefanv as with all of these tick marks, they are at the reviewers' discretion. =) The idea should certainly be considered for all PRs. And the idea is to build a suite over time. So anything that touches e.g. dilation, gaussian filters, watershed, etc should add a benchmark, and that benchmark should be run before and after the change, so that skimage moves towards speed as a whole.